teacher.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person_class'
+
+class Student < Person
+  def intialize(classroom)
+    @classroom = classroom
+    super(
+      age,
+      name,
+      parent_permission
+    )
```

Comment:
- Now, we can update the super like this ` super(age, name)` then instantiating the instance variable like this `    @specialization = specialization`